### ISSUE TITLE
Add content block menu not showing in page when using Entity/Item Picker field (e.g. Accordion App)

### ISSUE THREAD

**Issue by gfrater** [2sic/2sxc#2908]
**I'm submitting a ...**
[x] bug report

**...about**  
[x] edit experience / UI

**Current behavior**   
Using the Accordion App with the 2shineBS5 theme (also applies to any App that has and Entity/Item Picker field) shows the add content block menu correctly within the accordion itself but stops the add content block menu from showing in the rest of the page

**Expected behavior**   
The add content block menu should show both within the accordion (with the lighter blue buttons) and in the page (with the darker blue buttons)

**Instructions to Reproduce the Problem**
On a blank page add the acccordion app (or any app with an Entity/Item Picker field)

**Why change the behavior?**  
The current workaround is to put the page into edit mode and add a 2sxc Content or 2sxc App module using the DNN add module buttons. This means the editing experience will be different between pages as in some cases you don't need to enter edit mode and in some you do.

**Your environment**
* **2sxc version(s):** 14.07.04, 14.10.0
* **Browser:** all
* **DNN:** 9.10.2, 9.11.0 
* **Language:** English

**Anything you would like to add**

**Comment by iJungleboy** [comment]
I did a quick test - not the very latest 2shine but still a fairly recent version. I cannot reproduce the problem. From what I understand, the dark-blue bar wouldn't appear in your case. It does for me:

![image](https://user-images.githubusercontent.com/5321186/198575644-9a921c61-b1f4-460c-9834-772b072ad681.png)

Could it be that something else on the page (another module, maybe one in the accordion) is changing the behavior?

Basically it's controlled through various json nodes in html attributes - see https://docs.2sxc.org/js-code/quick-e/index.html

Could you see which ones your page has, and what is creating them?

**Comment by iJungleboy** [comment]
Believe it's not a 2sxc issue, will close. Add more infos if we should re-open.
